portfolio/src/main/webapp/mapScript.js

```diff
@@ -11,10 +11,159 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-
+let map;
+let editMarker
```

Comment:
missing semicolon

Reply:
More a question than a review, but what's the difference of declaring variables with let and var? I've usually thought of let as a function programming specific thing.